.gitignore

```diff
@@ -1,14 +1,14 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # dependencies
-/node_modules
+node_modules
 
 # testing
-/coverage
+coverage
 
 # production
-/build
-/dist
+build
+dist
```

Comment:
Minor thing that shouldn't change how the ignore works: add a trailing slash to the things we expect to be directories. Git will ignore at any depth as long as there's no leading or middle slash, but trailing should be fine